docker-orchestration/osrm-backend/README.md

```diff
@@ -53,8 +53,32 @@ total 243M
 $ cd ..
 
 # pull & run
-$ docker pull telenavmap/osrm-backend:no.63-20191112-master-telenav-845f953
-$ docker run -d -p 5000:5000  "src=$(pwd)/osrm-data,dst=/osrm-data,type=bind" telenavmap/osrm-backend:no.63-20191112-master-telenav-845f953 routed_no_traffic_startup 
+$ docker pull telenavmap/osrm-backend
+$ docker run -d -p 5000:5000 --mount "src=$(pwd)/osrm-data,dst=/osrm-data,type=bind" telenavmap/osrm-backend routed_no_traffic_startup 
+5b54931c035abaa0d0635cae4539da91e91fca02d1b37426451aa73476dd53fd
```

Comment:
This line indicate for `docker container ID`, right?

Reply:
yes